dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -881,6 +886,49 @@ public List<User> updateUserMetadata(@ApiParam(hidden = true) @Parameter(hidden
         return userDAO.findAll();
     }
 
+    @POST
+    @UnitOfWork
+    @RolesAllowed("admin")
+    @Path("/updateUserWorkflows")
+    @Operation(operationId = "checkWorkflowOwnership", description = "Check workflow ownership", security = @SecurityRequirement(name = JWT_SECURITY_DEFINITION_NAME))
+    @ApiResponse(responseCode = HttpStatus.SC_NO_CONTENT
+        + "", description = "Successfully updated workflow ownership for all users", content = @Content(array = @ArraySchema(schema = @Schema(implementation = User.class))))
```

Comment:
`content` may not be set correctly here.

Reply:
How embarrasing.